e2e-tests/.gitignore

```diff
@@ -67,4 +67,12 @@ site
 .idea
 
 # e2e test results
-junit-results.xml
\ No newline at end of file
+junit-results.xml
+/test-results/
+/playwright-report/
+/blob-report/
+/playwright/.cache/
+/test-results/
+/playwright-report/
+/blob-report/
+/playwright/.cache/
```

Comment:
```suggestion
```

redundant gitignores please remove. these are already there in `e2e-tests/.gitignore` : https://github.com/janus-idp/backstage-showcase/pull/1320/files#diff-7afc14328ee3ba71c8e2b91ab733c1edf4721523870b51d783e3ad7ab5d0096dR55-R64